testing/integration_test.go

```diff
@@ -231,7 +198,64 @@ func TestAgentIntegration(t *testing.T) {
 			nodeVersion:  "10",
 		},
 	}
+	if !enableV8Canary {
```

Comment:
I had this comment below before, I don't think it was addressed or responded.

This and below code can be a bit shorter as:
```
if !enableV8CanaryTest {
  testcases = append(testcases, nodeGCETestCase{
    ...
  })
}
return testcases
```

You can also then keep the test case list in the original test function, no need for gceTestcases I think.

Reply:
Done.